idseq/uploader.py

```diff
@@ -258,14 +260,34 @@ def get_user_agreement():
 class Tqio(io.BufferedReader):
     def __init__(self, file_path, i, count):
         super(Tqio, self).__init__(io.open(file_path, "rb"))
-        desc = "%s (%d/%d)" % (file_path, i + 1, count)
-        self.tqdm = tqdm.tqdm(
-            desc=desc,
-            unit="bytes",
-            unit_scale=True,
-            total=os.path.getsize(file_path))
+        curses.initscr()
+        curses.curs_set(0)
```

Comment:
Does this require a terminal?   Will it work through ssh session that has no interactive terminal attached?

Reply:
In my experience terminal screen stuff is hard to debug when it fails.